apps/dcbustimes/dc_bus_times.star

```diff
@@ -0,0 +1,257 @@
+                child = render.Column(
+                    children = [
+                        Bus[0],
+                        Bus[1],
+                        Bus[2],
+                        Bus[3],
+                    ],
+                ),
+            ),
+        )
+    elif numPredictions == 5:
+        return render.Root(
+            delay = 500,
+            child = render.Marquee(
+                scroll_direction = "vertical",
+                height = 32,
+                align = "start",
+                offset_start = 2,
+                offset_end = 32,
+                child = render.Column(
+                    children = [
+                        Bus[0],
+                        Bus[1],
+                        Bus[2],
+                        Bus[3],
+                        Bus[4],
+                    ],
+                ),
+            ),
+        )
+    elif numPredictions == 6:
+        return render.Root(
+            delay = 500,
+            child = render.Marquee(
+                scroll_direction = "vertical",
+                height = 32,
+                align = "start",
+                offset_start = 2,
+                offset_end = 32,
+                child = render.Column(
+                    children = [
+                        Bus[0],
+                        Bus[1],
+                        Bus[2],
+                        Bus[3],
+                        Bus[4],
+                        Bus[5],
+                    ],
+                ),
+            ),
+        )
+    elif numPredictions == 7:
+        return render.Root(
+            delay = 500,
+            child = render.Marquee(
+                scroll_direction = "vertical",
+                height = 32,
+                align = "start",
+                offset_start = 2,
+                offset_end = 32,
+                child = render.Column(
+                    children = [
+                        Bus[0],
+                        Bus[1],
+                        Bus[2],
+                        Bus[3],
+                        Bus[4],
+                        Bus[5],
+                        Bus[6],
+                    ],
+                ),
+            ),
+        )
+    else:
+        return render.Root(
+            delay = 500,
+            child = render.Marquee(
+                scroll_direction = "vertical",
+                height = 32,
+                align = "start",
+                offset_start = 2,
+                offset_end = 32,
+                child = render.Column(
+                    children = [
+                        Bus[0],
+                        Bus[1],
+                        Bus[2],
+                        Bus[3],
+                        Bus[4],
+                        Bus[5],
+                        Bus[6],
+                        Bus[7],
+                    ],
+                ),
+            ),
+        )
+
+def GetTimes1(stopID, apiKey):
+    cached = cache.get("arrTimes1")
```

Comment:
Since StopID is included in the API request, I'm guessing the response body is specific to a single stop? If so, then the cache key probably also needs to include the StopID to make sure later calls to `GetTime1` aren't just returning the cached data from previous calls with (potentially) different StopID.